addons/spark/0.x/spark.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: spark
+  labels:
+    kubeaddons.mesosphere.io/name: spark
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    stage.kubeaddons.mesosphere.io/spark: Beta
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.1.0-2"
+    appversion.kubeaddons.mesosphere.io/spark: "0.2.0"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.16.0
+  cloudProvider:
```

Comment:
What about `azure` and `gcp`? The default will be disabled of course, but it's good to be explicit and add them here.